### DISCUSSION TITLE
Rule for "How to include new pull requests"?

### DISCUSSION THREAD

**Discussion by astridx** [maplibre/maplibre-gl-js#484] [General]
Is it documented somewhere how exactly we include new pull requests? I always have a bad feeling about merging my own pull request, because I am used to the fact from another project that you are never allowed to merge your own.

**Comment by wipfli** [comment]
Squash and merge is the only allowed option, or are you referring to something else?

**Reply by HarelM** [reply]
If the review passes there's no problem in merging your own :-)
